Dockerfile-8.0

```diff
@@ -101,30 +74,50 @@ RUN apt-get -yqq install \
         php8.0-xml \
         php8.0-yaml \
         php8.0-zip \
-        newrelic-php5 \
-        newrelic-php5-common \
-        newrelic-daemon \
+        # below required to compile non-packaged extensions
+        php8.0-dev \
+        pkg-config \
+        libmemcached-dev \
+        zlib1g-dev \
+        libssl-dev \
+        libpcre3-dev \
+        golang \
     && \
-    /bin/bash /clean.sh \
+    /bin/bash -e /scripts/install-newrelic-ubuntu.sh && \
+    phpenmod \
+        memcached \
+        igbinary \
+        msgpack \
+    && \
+    phpdismod \
+        pdo_pgsql \
+        pgsql \
+        redis \
+        yaml \
+        xdebug \
     && \
-    phpenmod memcached && \
-    phpenmod igbinary && \
-    phpenmod msgpack && \
-    phpdismod pdo_pgsql && \
-    phpdismod pgsql && \
-    phpdismod redis && \
-    phpdismod yaml && \
-    phpdismod xdebug && \
+    curl -sS https://getcomposer.org/installer | php && \
```

Comment:
https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md let's add this script to install composer instead of this. The checksum part is the piece that really interests me for security sake.

Reply:
done